modules/mw.cx.MachineTranslationService.js

```
@@ -148,6 +151,12 @@ mw.cx.MachineTranslationService.prototype.fetchProviders = function () {
 		$to: this.targetLanguage
 	});
 
+	// {"mt":["MinT","Google"]}
+	// 2024 By: Ibrahem Qasim
+	if (this.targetLanguage === "or") {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)